cluster/gce/gci/configure-helper.sh

```diff
@@ -980,6 +1010,7 @@ function start-kube-controller-manager {
   if [[ -n "${CA_KEY:-}" ]]; then
     params+=" --cluster-signing-cert-file=/etc/srv/kubernetes/ca.crt"
     params+=" --cluster-signing-key-file=/etc/srv/kubernetes/ca.key"
+    params+=" --insecure-experimental-approve-all-kubelet-csrs-for-group=node-bootstrapper"
```

Comment:
I don't have a good sense for how these scripts are used (test vs dev vs production, etc)... is everyone that sets up a cluster using this expecting this to be on?